docs/.vuepress/sidebar.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
         {
           title: 'Front end code',
           children: [
-            [ 'guide/front-end-assets.md', 'Front end CSS and JS' ],
+            [ 'guide/front-end-assets.md', 'Front end CSS, JS and assets' ],
```

Comment:
This is the beginner guide section. The public directory stuff should be in an advanced section. If you add the new markdown page for it I can deal with the sidebar structure part. This is advanced because the beginner version would be to simply put stuff in the root public directory.

Reply:
Titled something like "Using static module assets"